api/nakadi-event-bus-api.yaml

```diff
@@ -568,6 +584,8 @@ paths:
 
         Calculate the distance between two offsets. This is useful for performing checks for data completeness, when
         a client has read some batches and wants to be sure that all delivered events have been correctly processed.
+
+        If per-EventType authorization is enabled, the called must be authorized to read from the EventType.
```

Comment:
This should be "caller" instead of "called", no? There are many occurrences of this in this document.